es/index.js.flow

```diff
@@ -0,0 +1,201 @@
+/* @flow */
+import React from 'react';
+import {withRouter} from 'react-router-dom';
+import type {HistoryAction, Location, Match, RouterHistory} from 'react-router-dom';
+
+declare type PropsT = {
+  afterCancel?: Function,
+  afterConfirm?: Function,
+  beforeCancel?: Function,
+  beforeConfirm?: Function,
+  children: (data: {isActive: bool, onCancel: Function, onConfirm: Function}) => React$Element<*>,
+  match: Match,
+  history: RouterHistory,
+  location: Location,
+  renderIfNotActive: bool,
+  when: bool | (Location, ?Location) => bool,
+  disableNative: bool,
+  allowGoBack: bool,
```

Comment:
These typings should also be updated to be more useful (as it is done in ts typings):

```
  allowGoBack?: boolean;
  renderIfNotActive?: boolean;
  disableNative?: boolean;
```

but I think it would be better to do it in the separate PR